modules/tinymce/src/core/test/ts/browser/EditorCleanupTest.ts

```diff
@@ -1,73 +1,67 @@
-import { Assertions, Chain, Log, NamedChain, Pipeline } from '@ephox/agar';
-import { Assert, UnitTest } from '@ephox/bedrock-client';
-import { Arr } from '@ephox/katamari';
+import { before, describe, it } from '@ephox/bedrock-client';
+import { Fun } from '@ephox/katamari';
 import { Editor as McEditor } from '@ephox/mcagar';
 import { Attribute, Remove, SugarElement, Truncate } from '@ephox/sugar';
+import { assert } from 'chai';
 
 import Editor from 'tinymce/core/api/Editor';
 import { RawEditorSettings } from 'tinymce/core/api/SettingsTypes';
 import VisualBlocksPlugin from 'tinymce/plugins/visualblocks/Plugin';
 import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.core.EditorCleanupTest', (success, failure) => {
-  Theme();
-  VisualBlocksPlugin();
+const assertPageLinkPresence = (url: string, exists: boolean): void => {
+  const links = document.head.querySelectorAll(`link[href="${url}"]`);
+  assert.equal(exists, links.length > 0, `Should have link with url="${url}"`);
```

Comment:
This assertion reads really strangely, and I think `expected` and `actual` are flipped here if I'm reading it correctly?

Reply:
You're right they are backwards.

Reply:
Fixed